### ISSUE TITLE
Delete button on ActionBar lacks information to screen reader users

### ISSUE THREAD

**Issue by allinox** [Altinn/altinn-access-management-frontend#363]
### Description of the bug

For users of screen readers, the delete button gives little to no information on what will actually be deleted should they press it.

Suggested fix is to add an aria-label that points to the title of the element that is to be deleted. This reference won't be visual, but it will be read out loud by the screen reader when regarding the button

![image](https://github.com/Altinn/altinn-access-management-frontend/assets/22012997/de5c9672-d410-4461-a4d0-876d3a3ab913)

### Steps To Reproduce

You can find this on the Overview page.
